src/datatypes/src/schema/constraint.rs

```diff
@@ -164,22 +164,23 @@ fn create_current_timestamp_vector(
 ) -> Result<VectorRef> {
     // FIXME(yingwen): We should implements cast in VectorOp so we could cast the millisecond vector
     // to other data type and avoid this match.
-    match data_type {
-        ConcreteDataType::Timestamp(_) => Ok(Arc::new(TimestampMillisecondVector::from_values(
-            std::iter::repeat(util::current_time_millis()).take(num_rows),
-        ))),
-        ConcreteDataType::Int64(_) => Ok(Arc::new(Int64Vector::from_values(
-            std::iter::repeat(util::current_time_millis()).take(num_rows),
-        ))),
-        _ => error::DefaultValueTypeSnafu {
+    let current_timestamp_vector = TimestampMillisecondVector::from_values(
```

Comment:
suggestion: Could you please add more tests for this method?

You could write a test similar to `test_create_default_vector_by_func` but checks that we can create default values for timestamp type with other resolutions (e.g. seconds)
https://github.com/GreptimeTeam/greptimedb/blob/81ea61ba43f5f62bb509cdbb58e1b779c1b5a1f4/src/datatypes/src/schema/constraint.rs#L261-L273